src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -312,6 +312,9 @@
     type: string
     short-summary: How outbound traffic will be configured for a cluster.
     long-summary: Select between loadBalancer and userDefinedRouting. If not set, defaults to type loadBalancer. Requires --vnet-subnet-id to be provided with a preconfigured route table and --load-balancer-sku to be Standard.
+  - name: --auto-upgrade-channel
+    type: string
+    short-summary: Specify the upgrade channel for autoupgrade. It could be rapid, stable, patch, node-image or none, none means disable autoupgrade.
```

Comment:
>  c.argument('auto_upgrade_channel', arg_type=get_enum_type([CONST_RAPID_UPGRADE_CHANNEL, CONST_STABLE_UPGRADE_CHANNEL, CONST_PATCH_UPGRADE_CHANNEL, CONST_NODE_IMAGE_UPGRADE_CHANNEL, CONST_NONE_UPGRADE_CHANNEL]))

Because you have defined the possible value of enumeration, so there is no need to prompt for possible values. These information will be automatically added to help

Reply:
This looks like its the help text for the full create command. You sure this would be automatically inserted? That does not seem to be the style that is followed.

Reply:
Yes, the enumerated values will be automatically inserted at the end of the parameter help description.
Such as:
```Python
Allowed values: CONST_RAPID_UPGRADE_CHANNEL, CONST_STABLE_UPGRADE_CHANNEL......
```

Reply:
Therefore, I suggest that you do not define enumeration separately and do not write the possible values in help. You just need to define the valid values directly in `arg_type=get_enum_type()`